src/foam/core/Property.js

```diff
@@ -287,7 +287,12 @@ foam.CLASS({
     },
     {
       name: 'type',
-    }
+    },
+    {
+      class: 'Boolean',
+      name: 'isSortable',
```

Comment:
I would say 'is' is not necessary, as 'ble' seems like a verb as well.

Reply:
fixed